samples/detect.v1p2beta1.js

```diff
@@ -24,31 +24,38 @@ function detectPdfText(bucketName, fileName) {
   // Creates a client
   const client = new vision.ImageAnnotatorClient();
 
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const bucketName = 'Bucket where the file resides, e.g. my-bucket';
+  // const fileName = 'Path to PDF file within bucket, e.g. path/to/document.pdf';
```

Comment:
Teeny nit: typically the format we use for this is to put the value description _above_ the value itself, e.g:

```
// Bucket where the file resides
// const bucketName = 'my-bucket';
```

Reply:
Sure. I will make the change.